src/components/Onboarding/Screens/Apps/IssuanceSettings/index.js

```diff
@@ -187,8 +187,8 @@ function IssuanceSettings() {
           `}
         >
           The initial ratio refers to the ratio of tokens to be minted and sent
-          to the Common Pool. These tokens will then be available to be
-          distributed through conviction voting.
+          to the Common Pool. These will then be available to be distributed
+          through conviction voting.
```

Comment:
same here, I don't think this change is related to this PR in anyways.

Reply:
Same as above